Marlin\src\module\temperature.cpp:826:46: error: operator ',' has no right operand[BUG] (short description)

When I try to compile the current Marlin 2.0_bugfix with #define PID_DEBUG  enabled i get the following compile error: 
Marlin\src\module\temperature.cpp:826:46: error: operator ',' has no right operand

from 
https://github.com/MarlinFirmware/Marlin/blob/09eb30dc05a727e3816ab5a407f02ada18700e23/Marlin/src/module/temperature.cpp#L821-L829

```cpp
      #if DISABLED(PID_OPENLOOP)
        SERIAL_ECHOPAIR(
          MSG_PID_DEBUG_PTERM, work_pid[HOTEND_INDEX].Kp,
          MSG_PID_DEBUG_ITERM, work_pid[HOTEND_INDEX].Ki,
          MSG_PID_DEBUG_DTERM, work_pid[HOTEND_INDEX].Kd
          #if ENABLED(PID_EXTRUSION_SCALING),
            MSG_PID_DEBUG_CTERM, work_pid[HOTEND_INDEX].Kc
          #endif
        );
```

I tried to remove the , after `#if ENABLED(PID_EXTRUSION_SCALING)` and add it to `MSG_PID_DEBUG_DTERM, work_pid[HOTEND_INDEX].Kd` but this causes additional errors.

Edit: I use the latest VSCode with PlatformIO

--- a/Marlin/src/lcd/ultralcd.h
+++ b/Marlin/src/lcd/ultralcd.h
@@ -187,6 +187,7 @@
 
 #else
 
+  #undef BUTTON_EXISTS
   #define BUTTON_EXISTS(BN) false
 
   // Shift register bits correspond to buttons:

--- a/Marlin/src/module/temperature.cpp
+++ b/Marlin/src/module/temperature.cpp
@@ -824,8 +824,7 @@ float Temperature::get_pid_output(const int8_t e) {
           MSG_PID_DEBUG_ITERM, work_pid[HOTEND_INDEX].Ki,
           MSG_PID_DEBUG_DTERM, work_pid[HOTEND_INDEX].Kd
           #if ENABLED(PID_EXTRUSION_SCALING)
-            ,
-            MSG_PID_DEBUG_CTERM, work_pid[HOTEND_INDEX].Kc
+            , MSG_PID_DEBUG_CTERM, work_pid[HOTEND_INDEX].Kc
           #endif
         );
       #endif

--- a/Marlin/src/module/temperature.cpp
+++ b/Marlin/src/module/temperature.cpp
@@ -823,7 +823,8 @@ float Temperature::get_pid_output(const int8_t e) {
           MSG_PID_DEBUG_PTERM, work_pid[HOTEND_INDEX].Kp,
           MSG_PID_DEBUG_ITERM, work_pid[HOTEND_INDEX].Ki,
           MSG_PID_DEBUG_DTERM, work_pid[HOTEND_INDEX].Kd
-          #if ENABLED(PID_EXTRUSION_SCALING),
+          #if ENABLED(PID_EXTRUSION_SCALING)
+            ,
             MSG_PID_DEBUG_CTERM, work_pid[HOTEND_INDEX].Kc
           #endif
         );

